src/Routing.js

```diff
@@ -12,20 +13,13 @@ const Routing = () => {
 
   return (
     <Router>
+      <NavBar />
       <Switch>
-        {signedOutRoutes.map(
-          ({Component, path}, index) => {
-            return (
-              <Route
-                key={index}
-                exact
-                path={path}
-                component={Component}
-              />
-            );
-          }
-        )}
-        <Redirect  to='/'/>
+        {signedOutRoutes.map(({ Component, path }, index) => {
+          return  (
+            <Route key = { index } exact path = { path } component = { Component } />);
```

Comment:
put the attributes on multiples lines please. don't put empty spaces next to these; this is a dynamic rendering thing; the imports at the top using curly braces do need empty spaces